cupyx/fallback_mode/utils.py

```diff
@@ -41,3 +44,78 @@ def _call_numpy(func, args, kwargs):
     cupy_res = data_transfer._get_cupy_result(numpy_res)
 
     return cupy_res
+
+
+_thread_locals = threading.local()
+_thread_locals.dispatch_type = 'warn'
+
+FallbackWarning = type('FallbackWarning', (Warning,), {})
+warnings.simplefilter(action='always', category=FallbackWarning)
+
+
+def geterr():
+    return _thread_locals.dispatch_type
+
+
+def seterr(new_dispatch):
+
+    old = _thread_locals.dispatch_type
+
+    if new_dispatch in ['print', 'warn', 'log', 'ignore', 'raise']:
+        _thread_locals.dispatch_type = new_dispatch
+        return old
+
+    raise ValueError('{} is not valid dispatch type'.format(new_dispatch))
+
+
+class FallbackLogger:
+
+    logger = None
+
+    @classmethod
+    def setlogger(cls, logger):
+        cls.logger = logger
+
+    @classmethod
+    def getlogger(cls):
+
+        if cls.logger is None:
+            raise AttributeError('Logger not initiated')
```

Comment:
I can't seem to decide appropriate error here. So, for now ended up with `AttributeError`.